test/unit/config/config-location.test.js

```diff
@@ -45,11 +45,6 @@ tap.test('when overriding the config file location via NR_HOME', (t) => {
     })
   })
 
-  /**
-   * TODO: Replace `rmdir` with `rm` when drop Node 12.
-   * `rmdir` has been deprecated but preferred `rm` was introduced in Node 14.
-   * https://nodejs.org/api/deprecations.html#DEP0147
-   */
```

Comment:
@bizob2828 We seem to have a contradiction here. This TODO says that `rm` is preferred for Node versions newer than 12, with `rmdir` being deprecated. The other comment in versioned-external, however, says the opposite.

Reply:
Agreed.

Reply:
https://nodejs.org/api/fs.html#fspromisesrmdirpath-options says 

`To get a behavior similar to the rm -rf Unix command, use [fsPromises.rm()](https://nodejs.org/api/fs.html#fspromisesrmpath-options) with options { recursive: true, force: true }.`

Reply:
Those are the Node 18 docs, so...`rm` seems like it should be preferred.

Reply:
yea just remove the comments, but also update the `require('fs').promises` to be `require('fs/promises')`